### ISSUE TITLE
Failure on organization repo with Not Found

### ISSUE THREAD

**Issue by rachel-fenichel** [google-github-actions/release-please-action#414]
### TL;DR

I set up release-please on my personal fork of a repo to make sure it worked (it did) and then applied the same settings to the original repository, which is in the google organization, and it now fails with "release-please failed: Not Found".

### Expected behavior

Release-please runs and creates a PR.

### Observed behavior

Release-please correctly runs on merges to develop, but fails with "release-please failed: Not Found".

### Action YAML

```yaml
on:
  push:
    branches:
      - develop
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: GoogleCloudPlatform/release-please-action@v3.0.0
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          release-type: node
          package-name: blockly
          command: manifest
```
```


### Additional information

Here's the [PR that created the release-please config](https://github.com/google/blockly/pull/5848).

I set it up with a manifest for bootstrapping purposes. That's one difference between my runs here and on my personal fork: I used a different SHA since my fork was out of date.

As far as I can tell, the `token:`, `release-type:`, and `package-name:` properties in the action YAML should be redundant or overwritten by the manifest, but I kept them since they were the values I used during debugging on my fork.

**Comment by rachel-fenichel** [comment]
It turned out the issue was that my organization has stricter permissions for the default github token, which meant the action could not create a pull request. I switched to using it as a bot, installed on my organization and configured for my repository, and now it works.
